include/device.h

```diff
@@ -286,6 +286,16 @@ struct device *device_get_binding(const char *name);
  */
 #define DEVICE_PM_SUSPEND_STATE         3
 
+/** @def DEVICE_PM_FORCE_SUSPEND_STATE
+ *
+ * @brief device is in SUSPEND power state
+ *
+ * @details Most device context is lost by the hardware.
+ * Device drivers must save and restore or reinitialize any context
+ * lost by the hardware
+ */
+#define DEVICE_PM_FORCE_SUSPEND_STATE	4
```

Comment:
What is the actual difference with above DEVICE_PM_SUSPEND_STATE? you copy-pasted the description of that one without detailing that difference.

Reply:
In case suspend call, the driver can return error based on the device state like it is middle of some transaction etc..but in case force suspend the device has to be put in suspend by canceling or terminating any ongoing transactions.